src/commands/test.js

```diff
@@ -13,8 +13,7 @@ const test = (context) => {
     extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
   }
 
-  if (process.version !== LAMBDA_VERSION) {
-    context.line(`You're running tests on Node ${process.version}, but Zapier runs your code on ${LAMBDA_VERSION}. The version numbers must match. See https://zapier.github.io/zapier-platform-cli/index.html#requirements for more info.`);
+  if (!utils.correctVersion()) {
```

Comment:
I figured that was useful for being able to use `context.line` instead of `console.log`, but I may be underestimating. 

Happy to rename, that's probably a good way to signal it just returns a bool. Also thought about moving `process.exit_status = 1` into the function itself, but wasn't sure if we wanted the side effect in there.